packages/cli/src/utils/settingsUtils.ts

```diff
@@ -431,11 +446,9 @@ export function getDisplayValue(
   const isChangedFromDefault =
     typeof defaultValue === 'boolean' ? value !== defaultValue : value === true;
   const isInModifiedSettings = modifiedSettings.has(key);
-  const hasPendingChanges =
-    pendingSettings && settingExistsInScope(key, pendingSettings);
 
-  // Add * indicator when value differs from default, is in modified settings, or has pending changes
-  if (isChangedFromDefault || isInModifiedSettings || hasPendingChanges) {
+  // Only show * if value actually differs from default, not just if key exists
```

Comment:
is that right? a value that matches the default is still crucial to show if it is a workspace level setting that overrides the value in the user settings.